x-pack/plugins/security_solution/cypress/tasks/rule_details.ts

```diff
@@ -99,9 +99,9 @@ export const goToExceptionsTab = () => {
 };
 
 export const editException = () => {
-  cy.get(EXCEPTION_ITEM_ACTIONS_BUTTON).click();
```

Comment:
The `Displays version conflict error` and `Displays missing exception list error` tests in `exceptions_flyout.spec.ts` invoke the `editException()` test helper.

This `editException()` helper function was added ~ 4 days ago, [via this PR](https://github.com/elastic/kibana/pull/135255/files#diff-e184b6d6539304e06fe6c7cdaa63773a60678e792d2c1b27a9873977c5a4e64bR101) and is failing on CI with the following error:

```
CypressError: Timed out retrying after 120050ms: `cy.click()` failed because the center of this element is hidden from view:

`<button class="euiButtonIcon euiButtonIcon--primary euiButtonIcon--empty euiButtonIcon--xSmall" type="button" aria-label="Exception item actions menu" data-test-subj="exceptionItemCardHeader-actionButton">...</button>`

Fix this problem, or use `{force: true}` to disable error checking.
```

The test passes locally. I'm implementing the fix suggested in the error message above.